starter-code/src/viking.js

```diff
@@ -1,11 +1,79 @@
 // Soldier
-function Soldier () {}
-
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function() {
+  return this.strength;
+};
+Soldier.prototype.receiveDamage = function(damage) {
+  this.health = this.health - damage;
```

Comment:
Aquí también podrías haber hecho esto:
this.health -= damage;
Y escribes menos código.